packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -34,6 +37,8 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
     super.attach(owner);
     game.attach(owner, this);
 
+    final gameLoop = this.gameLoop = GameLoop(gameLoopCallback);
```

Comment:
No need for this variable right? You can just set `this.gameLoop` and that will be in the scope for this method?
```suggestion
    this.gameLoop = GameLoop(gameLoopCallback);
```

Reply:
Analyzer is dumb, not recnizing the non nullability there.

Reply:
Yeah, this was with the other comment in mind.
But then I guess this can be merged if you just run for formatter on it then? :)